catalog/dags/database/staging_database_restore/staging_database_restore_dag.py

```diff
@@ -76,7 +77,8 @@
 def restore_staging_database():
     # If the `recreate_full_staging_index` DAG was manually triggered prior
     # to the database restoration starting, we should wait for it to
-    # finish.
+    # finish. It is not necessary to wait on any of the other ES DAGs as
+    # they do not directly affect the database.
     wait_for_recreate_full_staging_index = wait_for_external_dag(
         external_dag_id=RECREATE_STAGING_INDEX_DAG_ID,
     )
```

Comment:
So because this is not using the `prevent_concurrency_with_dags_with_tag` utility, the DAG for restoring staging DB will not wait for other DAGs except for the one that is explicitly specified here, right?

Reply:
Accurate, but this has now been modified to instead use `wait_for_external_dags_with_tag` and the new staging db concurrency group 👍